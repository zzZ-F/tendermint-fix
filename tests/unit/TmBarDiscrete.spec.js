import { shallowMount } from "@vue/test-utils"
import htmlBeautify from "html-beautify"
import { TmBarDiscrete } from "@/index"

describe("TmBarDiscrete.vue", () => {
  let wrapper
  beforeEach(async () => {
    const nodes = ["asdf", "sadf"]
    const active = 0
    const clickFn = function(e) {
      return e
    }
    wrapper = shallowMount(TmBarDiscrete, {
      propsData: { nodes, active, clickFn }
    })
  })

  it("has the expected html structure", () => {
    expect(htmlBeautify(wrapper.html())).toMatchSnapshot()
  })
})
