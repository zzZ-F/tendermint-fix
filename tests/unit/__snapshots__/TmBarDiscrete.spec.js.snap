// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TmBarDiscrete.vue has the expected html structure 1`] = `
<div class="tm-bar-discrete">
  <div class="tm-bar-discrete__container">
    <div class="tm-bar-discrete__edge"></div>
    <div class="tm-bar-discrete__node tm-bar-discrete__node--active"></div>
    <div class="tm-bar-discrete__node"></div>
  </div>
</div>
`;
